dask_kubernetes/core.py

```diff
@@ -207,7 +207,22 @@ async def start(self, **kwargs):
             self.external_address = "tcp://{host}:{port}".format(
                 host=loadbalancer_host, port=SCHEDULER_PORT
             )
-        # FIXME Set external address when using nodeport service type
+        if self.service.spec.type == "NodePort":
+            start = time.time()
+            hostname = dask.config.get("kuberntetes.scheduler-service-nodeport-host")
```

Comment:
Could you please set a default for this in the `kubernetes.yaml` configuration file.

Reply:
added to defaults, and fixed typo. in new commit